install/crd/patches/oneof_in_authconfigs.yaml

```diff
@@ -75,6 +75,22 @@
         kubernetes: {}
       required: [name, authzed]
 
+- op: add
+  path: /spec/versions/0/schema/openAPIV3Schema/properties/spec/properties/response/items/oneOf
+  value:
+    - properties:
+        name: {}
+        opa: {}
+      required: [name, wristband]
+    - properties:
+        name: {}
+        json: {}
+      required: [name, json]
+    - properties:
+        name: {}
+        kubernetes: {}
+      required: [name, plain]
```

Comment:
Oops. Nice catch! Thanks, @bartoszmajsak !

Reply:
@guicassolato I think it would be worth reviewing the whole file then. It seems to be copy-paste error.

Examples: 

- https://github.com/Kuadrant/authorino/blob/text-plain-response/install/crd/patches/oneof_in_authconfigs.yaml#L24-L25C29
- https://github.com/Kuadrant/authorino/blob/text-plain-response/install/crd/patches/oneof_in_authconfigs.yaml#L36-L40
- https://github.com/Kuadrant/authorino/blob/text-plain-response/install/crd/patches/oneof_in_authconfigs.yaml#L53-L56
- https://github.com/Kuadrant/authorino/blob/text-plain-response/install/crd/patches/oneof_in_authconfigs.yaml#L73-L76

Makes we wonder if that can be prevented going forward with some test automation.

Reply:
Yep. Lots of errors 😕 Thanks for pointing out.
I'll send a separate PR with fix and tests. Until then, I wonder if those OAS constraints are even working at all!

Reply:
Fixed in #396